pkg/crypto/age.go

```diff
@@ -104,6 +104,20 @@ func Identity() (*age.X25519Identity, error) {
 	return generateIdentity(getAgePath())
 }
 
+func findMissingKeyForEmail(emails []string, keys []*api.PublicKey) []string {
+	if len(keys) == 0 {
+		return emails
+	}
+	emailSet := sets.NewString(emails...)
+	emailKeySet := sets.NewString()
+	for _, key := range keys {
+		if key.User != nil && key.User.Email != "" {
+			emailKeySet.Insert(key.User.Email)
+		}
+	}
+	return emailSet.Difference(emailKeySet).List()
```

Comment:
Is there no empty function to call here and have this func be a bool?

Can also use the new set I implemented in our go lib, which has a set equality func

Reply:
done